### PR TITLE
feat(strm-2244): initial commit for cli support for privacy metrics

### PR DESCRIPTION
**Pull request by bobvandenhoogen**

### CODE REVIEW

File: pkg/diagnostics/diagnostics.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+const privacyDiagnosticsServiceUrl = "/privacy-diagnostics/upload"
+
+func evaluate(cmd *cobra.Command) {
+	flags := cmd.Flags()
+	dataFile := util.GetStringAndErr(flags, dataFileFlagName)
+	qi := getFlags(cmd, quasiIdentifierFlagName)
+	sa := getFlags(cmd, sensitiveAttributeFlagName)
+	metrics := getFlags(cmd, metricsFlagName)
+	//metrics, err := flags.GetStringArray(metricsFlagName)
```
Comment (bobvandenhoogen) [inline pkg/diagnostics/diagnostics.go:36]: commented code

File: pkg/diagnostics/diagnostics.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	"strmprivacy/strm/pkg/common"
+	"strmprivacy/strm/pkg/util"
+)
+
+type Metrics struct {
+	K int            `json:"k"`
+	L map[string]int `json:"L"`
+	T float64        `json:"T"`
+	E string         `json:"error"`
+}
```
Comment (ivan-p92) [inline pkg/diagnostics/diagnostics.go:26]: The names are really short, with the explicit json mapper they could be more descriptive. Does the type need to be public?

**Review by ivan-p92 — APPROVED** [review]

File: pkg/diagnostics/diagnostics.go (hunk 2 above)
Comment (bobvandenhoogen) [reply pkg/diagnostics/diagnostics.go:26]: type needs to be public. Can change the names

**Pull request merged**